### ISSUE TITLE
Add alt text to Search Topic icons

### ISSUE THREAD

**Issue by dneufeldcu** [cedardevs/onestop#268]
Search topic icons are missing alt text description.

**Comment by ajakz** [comment]
The landing page currently has aria-hidden on the search topic icons, and the screen reader is directed to read the text beneath the icons as buttons (i.e., the user would never know an image is there). Likewise, the icons on the search toolbar have descriptive text that is read by the screen reader. We'll discuss the accessibility design of the landing page along with a recording of the screen reader on the page to demonstrate during our large sprint review. Seems like one of the tools used to assess page accessibility will throw an error on any image, regardless of other aria tags on it, if it doesn't have alt-text.
